e2e/test-runner/go.mod

```
@@ -3,4 +3,7 @@ module github.com/ubuntu/wsl/e2e/test-runner
 go 1.18
 
 require github.com/ubuntu/wsl/e2e/constants v0.0.0
+
+require gopkg.in/ini.v1 v1.67.0 // indirect
+
```

We can do it in a single require clause:

```suggestion
require (
    github.com/ubuntu/wsl/e2e/constants v0.0.0
    gopkg.in/ini.v1 v1.67.0 // indirect
)
```